Lib/fontTools/ttLib/tables/_k_e_r_n.py

```diff
@@ -59,7 +61,7 @@ def compile(self, ttFont):
 			nTables = 0
 		if self.version == 1.0:
 			# AAT Apple's "new" format.
-			data = struct.pack(">ll", fl2fi(self.version, 16), nTables)
+			data = struct.pack(">LL", fl2fi(self.version, 16), nTables)
```

Comment:
the Apple spec has unsigned long integers (uint32) for version and nTables, not signed